src/Resources/config/routes.php

```diff
@@ -10,7 +10,6 @@
     $routes
         ->add('oauth2_authorize', '/authorize')
         ->controller([AuthorizationController::class, 'indexAction'])
-        ->methods(['GET'])
```

Comment:
is there a reason for restricting this route to `GET`? 

i'm asking because i want to show the user a consent form in this same route, without having to redirect to another page and return here.

this is currently doable if this route would allow other methods ( mainly `POST` ).

Reply:
I don't see any reason. I guess we can change it if we wanna add specific behavior. Maybe you can't add `POST` in the related PR?